Pascal's Triangle II.java

```diff
@@ -0,0 +1,46 @@
+/*
+Given a non-negative index k where k ≤ 33, return the kth index row of the Pascal's triangle.
+
+Note that the row index starts from 0.
+
+
+In Pascal's triangle, each number is the sum of the two numbers directly above it.
+
+Example:
+
+Input: 3
+Output: [1,3,3,1]
+Follow up:
+
+Could you optimize your algorithm to use only O(k) extra space?
+*/
+
+//This is using the formual of Binomial Coeffecient for the Pascal triangle
```

Comment:
dont use the formula
instead use the pattern given in the question

Reply:
Updated the Solution!